README.md

```diff
@@ -1,195 +1,19 @@
-# 🪐 NASA PDS New Java Project Template
+#### Registry Loader
```

Comment:
Maybe a single `#` to make the a `<h1>` header? It is the only header in the doc